librepo/yum.h

```diff
@@ -72,6 +85,72 @@ lr_yum_repo_free(LrYumRepo *repo);
 const char *
 lr_yum_repo_path(LrYumRepo *repo, const char *type);
 
+int
+hmfcb(void *clientp, const char *msg, const char *url);
```

Comment:
I think it's short for "handle mirror failure callback".  Yeah, I had to read the source for that before.